### DISCUSSION TITLE
Difference between <Image /> and <Img />

### DISCUSSION THREAD

**Discussion by connor-schratz** [chakra-ui/chakra-ui#7906] [Q&A]
Working on optimizing images on a Next site and while reading the docs I noticed the <Img /> component is recommended for usage with SSR. What is the difference between the two components? There wasn't a lot of information provided about how the two differ.

**Comment by gnowland** [comment]
Probably just

[Usage with SSR](https://v2.chakra-ui.com/docs/components/image#usage-with-ssr)
`import { Img } from '@chakra-ui/react'`
The `Img` Component provides a fallback for SSR applications which uses the image directly without client side checks.
